cg/meta/upload/scout/scout_config_builder.py

```diff
@@ -66,10 +66,9 @@ def add_common_sample_files(
         case_sample: CaseSample,
     ) -> None:
         """Add common sample files for different analysis types."""
-        sample_id: str = case_sample.sample.internal_id
```

Comment:
this has to be the internal case id

Reply:
This method is common to all pipelines, so we need to log the sample ID